packages/search-service/src/search-sitemap/resolver.ts

```diff
@@ -0,0 +1,23 @@
+import { Request } from 'express';
+import { SiteMaps } from './schema';
+
+export const getSiteMap = async (req: Request) => {
+  const { contentType } = req.query;
+
+  const SiteMapdata = await (
```

Comment:
`await` is not needed here as the return type of the `.map` is not a promise. Only 1 `await` is enough for the `Sitemaps.find`
```suggestion
  const SiteMapdata = (
```